include/wash.hpp

```diff
@@ -54,6 +54,25 @@
 #include "vector.hpp"
 #include "kernels.hpp"
 
+#define defineWashForceKernel(kernel_name, particle_name) \
+    void kernel_name (wash::Particle& particle_name, \
+    const std::vector<wash::Particle>::const_iterator& begin, \
```

Comment:
I think ideally these should be `const std::vector<wash::Particle>::const_iterator& begin`, so that the elements are `const` themselves

Reply:
Doesn't need to be `const` here I don't think. `const_iterator` enforces it (also throws an error if you try, because it'd try to make a `const const wash::Particle` I guess - which wouldn't exist) (https://en.cppreference.com/w/cpp/container/vector)